src/fs/driver/ramfs/ramfs_dvfs.c

```diff
@@ -0,0 +1,401 @@
+/**
+ * @file
+ * @brief Tmp file system
+ *
+ * @date 12.11.12
+ * @author Andrey Gazukin
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <limits.h>
+
+#include <fs/dvfs.h>
+
+#include <util/array.h>
+#include <util/indexator.h>
+#include <mem/misc/pool.h>
+#include <mem/phymem.h> /* PAGE_SIZE() */
+
+#include <util/math.h>
+#include <util/err.h>
+
+#include <embox/unit.h>
+#include <drivers/block_dev.h>
+
+/* define sizes in 4096 blocks */
+#define MAX_FILE_SIZE   OPTION_GET(NUMBER, ramfs_file_size)
+#define RAMFS_FILES     OPTION_GET(NUMBER, inode_quantity)
+#define FILESYSTEM_SIZE (MAX_FILE_SIZE * RAM_FILES)
+
+#define RAMFS_NAME_LEN	32
```

Comment:
May be use NAME_MAX from limits.h
[https://serverfault.com/questions/9546/filename-length-limits-on-linux](https://serverfault.com/questions/9546/filename-length-limits-on-linux)